CONTRIBUTING.md

```diff
@@ -16,46 +16,85 @@ In addition to Github issues, we also utilize a feedback site that helps our pro
 
 We have a 3 step process for contributions:
 
-1. Sign or be added to an existing [Contributor License Agreement (CLA)](https://supermarket.chef.io/become-a-contributor).
-2. Create a Github Pull Request for your change.
-3. Perform a [Code Review](#cr) with the project maintainers on the pull request.
+1. Commit changes to a git branch, making sure to sign those changes for the [Developer Certificate of Origin](#developer-certification-of-origin-dco).
```

Comment:
Sign means something different then sign-off, no?

Perhaps something like `Commit changes to a git branch, making sure to sign-off the changes under the Developer Certificate of Origin.`